.travis.yml

```diff
@@ -45,6 +45,13 @@ jobs:
       - REDUCE_SDK='True'
       script: ./scripts/ci/test_automation.sh
       python: 3.6
+    - stage: verify
+      env:
+      - PURPOSE='Automation'
+      - REDUCE_SDK='True'
+      - AZ_USE_CACHE='True'
```

Comment:
This value should also be able to come from the user's config file, which means the name needs to change to fit the expected format.